### ISSUE TITLE
stripe_color styling option throws an error in HTML mode

### ISSUE THREAD

**Issue by sam-crawley** [haozhu233/kableExtra#286]
If using the 'stripe_color' latex option in HTML mode, the following error is thrown:

`Error in htmlTable_styling(kable_input, bootstrap_options = bootstrap_options,  : 
  unused argument (stripe_color = "light-gray")`

It's fine if this is not supported in HTML (and other modes), but it would be good if it didn't throw an error so that the same code can be used for HTML & Latex.

**Comment by Mutant** [comment]
Tested in 1.0:
```
table <- kable(mtcars, format = "html")
table <- kable_styling(table, stripe_color = "light-gray")
````
Produces:
```
Error in htmlTable_styling(kable_input, bootstrap_options = bootstrap_options,  : 
  unused argument (stripe_color = "light-gray")
```
It would be nice if `htmlTable_styling()` ignored parameters relating to latex styling (and vice verca). Otherwise, in code where you want to be able to generate tables in either html or latex, you have to have a whole lot of:
`if (type == "html") { ... }` (etc.)

**Comment by haozhu233** [comment]
@themechanicalbear Thank you so much for your testing and comments!

@sam-crawley @Mutant As @themechanicalbear mentioned, I couldn't reproduce this error either. 

When I did
```
table <- kable(mtcars, format = "html")
table <- kable_styling(table, stripe_color = "light-gray")
table
```
I got a normally rendered table. 

<img width="648" alt="screen shot 2019-01-21 at 2 43 56 pm" src="https://user-images.githubusercontent.com/7014590/51495627-0dc22e00-1d8b-11e9-9d64-e415270d007c.png">

Also, you can check the source code. 
https://github.com/haozhu233/kableExtra/blob/5e1740df8766b6ee0a04810558f1b09da0635559/R/kable_styling.R#L123-L129

It doesn't take any `stripe_color` into consideration. 

If any of you can provide me a full example, it might help us locate this issue more accurately.

**Comment by sam-crawley** [comment]
It looks like this was fixed in 81eb9347 when the ellipsis was removed from the call to htmlTable_styling(). Even though I upgraded to 1.0 where this is fixed, for some reason 0.9 was sometimes getting loaded in my session (I have no idea why). 

It's definitely fixed in 1.0, so this can be closed.

Thanks for the help.

**Comment by haozhu233** [comment]
@sam-crawley Great! 🎉Glad to hear that this issue has already been fixed for you! 

@themechanicalbear Thanks again for looking into this case! Happy hacking! :)
